essen/faqs/models.py

```diff
@@ -0,0 +1,63 @@
+import colorsys
+import json
+
+import django.db.models
+from colorfield.fields import ColorField
+from django.core.validators import MaxValueValidator, MinValueValidator
+from django.db import models
+from django.utils.html import escape
+from django.utils.safestring import SafeString
+
+from essen.helper import reify, replace_url_with_link
+from faqs.units import units
+
+
+class Faq(models.Model):
+    question = models.TextField()
+    answer = models.TextField()
+    category = models.ForeignKey(
+        "faqs.Category",
+        on_delete=django.db.models.deletion.SET_NULL,
+        blank=True,
+        null=True,
+    )
+
+    def __str__(self):
+        return self.question
+
+    @property
+    def structured_data(self):
+        # https://schema.org/Faq
```

Comment:
The schema Faq doesn't exist. Because this isn't a recipe, I would just get rid of it and the corresponding feed as well.